undefined_syms.txt

```diff
@@ -1020,21 +594,9 @@ D_06006F38 = 0x06006F38;
 D_06006FD8 = 0x06006FD8;
 D_06009930 = 0x06009930;
 
-// ovl_En_Slime
-
-D_060004C0 = 0x060004C0;
-D_06000650 = 0x06000650;
-D_06000828 = 0x06000828;
-D_06000A10 = 0x06000A10;
-
-// ovl_En_Snowwd
-
-D_06000198 = 0x06000198;
-D_06001AA0 = 0x06001AA0;
-
 // ovl_En_Ssh
 
-D_06000304 = 0x06000304;
+D_06000304 = 0x06000304; // Must keep, see bug in EnSsh_Init
```

Comment:
With this I'm curious if they accidentally used the `anim` symbol from `object_st` (Skulltula), and if that is what we should do instead.  The two animations look similar (if not identical), and in OoT the animations were both at the same offset in their respective object files.

Reply:
As a precedent we had a similar issue with the `sPlayerFirstPersonRightHandDLs` and `sPlayerFirstPersonRightHandHookshotDLs` arrays of `z_player_lib.c`. We ended up leaving the hardcoded address there.
I don't have any strong opinion either way, but I kinda lean towards using an existing symbol, which would mean using the anim symbol from `object_st`

Reply:
(one thing I forgot to mention, is since this use is in code, hard coding a hex number does not work. It needs to be a symbol for it to match, which is why this exists)

Reply:
Yeah, I also don't feel strongly either way.

Reply:
Went ahead and made the change. If anyone has issues, or think I can improve the comment, just let me know.